lambda-notes/src/components/Authentication.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import Login from './Login'
+
+
+ const Authentication = App =>
+  class extends Component {
```

Comment:
you could give this anonymous class a name so when we look at the React dev tools it wont give this HOC a funky name like **<_class2>**